src/pykeen/training/callbacks.py

```diff
@@ -53,9 +53,12 @@ def pre_step(self, **kwargs: Any):
 
 from typing import Any, Collection, List, Optional, Union
 
+from class_resolver import HintOrType, OptionalKwargs
 from torch.nn.utils import clip_grad_norm_, clip_grad_value_
 
-from ..trackers import ResultTracker
+from ..evaluation import Evaluator, MetricResults, evaluate, evaluator_resolver
+from ..trackers import ResultTracker, tracker_resolver
+from ..typing import MappedTriples
 
 __all__ = [
     "TrainingCallbackHint",
```

Comment:
make sure `EvaluationCallback` is in the `__all__` so it gets put in the reference docs

Reply:
890c179